pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -338,6 +338,15 @@ func (ic *ifConfigurator) validateContainerPeerInterface(interfaces []*current.I
 		containerVeth.name)
 }
 
+// getInterceptedInterfaces is not supported on Windows.
+func (ic *ifConfigurator) getInterceptedInterfaces(
+	sandbox string,
+	containerNetNS string,
+	containerIFDev string,
+) (*current.Interface, *current.Interface, error) {
+	return &current.Interface{}, &current.Interface{}, errors.New("getInterceptedInterfaces is unsupported on Windows")
```

Comment:
maybe just return nil when the returned value is a pointer for error case
you haven't imported package `errors` so it should get compiling error

Reply:
Greate catch! I will update.